.buildkite/pipeline.linux.yml

```diff
@@ -0,0 +1,139 @@
+steps:
+  - label: "build-gnu-x86"
+    commands:
+     - cargo build --release
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
```

Comment:
I noticed that for the windows pipeline you also specify an "os" tag. Shouldn't we do the same for the linux pipeline? I can re-start the agent and add a linux tag to it.

Reply:
That would be great. On one hand, it's more consistent and we avoid targeting the wrong agents by mistake.

I'll add os tags here.

Reply:
I added the os=linux tag to the buildkite agents.